tests/tests_remote_default_remote.yml

```diff
@@ -27,6 +30,19 @@
     - name: Force all notified handlers to run at this point, not waiting for normal sync points
       meta: flush_handlers
 
+    - name: Check rsyslog.conf size
+      assert:
+        that: rsyslog_conf_line_count.stdout.0 | int <= 7
+
+    - name: Check file counts in rsyslog.d
+      assert:
+        that: rsyslog_d_file_count.matched >= 8
+
+    - name: Check rsyslog errors
+      command: systemctl status rsyslog
+      register: systemctl_result
+      failed_when: "'error' in systemctl_result.stderr"
```

Comment:
`or systemctl_result.rc != 0`

Reply:
Actually, the return value is 0...
Please see this comment and the commit in pr115.
https://github.com/linux-system-roles/logging/pull/115#issuecomment-627882173
https://github.com/linux-system-roles/logging/pull/115/files#diff-b66e58c3692b490cd622af3414c88ab1R75-R81

Reply:
Right - I get that in some error cases, `rc` will be `0` and there will be the string `error` somewhere in the output.  But for some error cases, maybe not even ones that you are expecting, it is possible that `systemctl status rsyslog` will return an non-zero exit code, and the string `error` might not be in the output in that case (e.g. rsyslog crashed for some reason before being able to write out the full error message).  In that case, rsyslog will have failed, but you will have masked the error by using `failed_when`.

Reply:
Thanks, @richm. Fixed. Also, rebased with master.

Reply:
Indeed - `or` was part of the suggested code :-)